### PR TITLE
Suparna_Assignment_22_10min+5Min

### PR DESCRIPTION
**Pull request by SuparnaNikam**
PalindromeNo
PalindromeString

### CODE REVIEW

File: src/suparnaNJune20/Assignment22/PalindromeString.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+			index++;
+		}
+		if (str1.equalsIgnoreCase(tempStr))
+			// or if ((str1.compareToIgnoreCase(tempStr)) != 0)
+			System.out.println(str1 + " is Palindrome String  ");
+		else
+			System.out.println(str1 + " is not Palindrome String ");
+	}
+
+	void checkPalindromeStringMethod2(String str1) {
```
Comment (VireshJJun20) [inline src/suparnaNJune20/Assignment22/PalindromeString.java:33]: Code looks good to me.

File: src/suparnaNJune20/Assignment22/PalindromeNo.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+Input: number = 56789
+Output : Given number 56789 is not palindrome.
+
+*/
+package suparnaNJune20.Assignment22;
+
+public class PalindromeNo {
+
+	void checkPalindromeNo(int num) {
```
Comment (VireshJJun20) [inline src/suparnaNJune20/Assignment22/PalindromeNo.java:17]: Neat and clean code.

File: src/suparnaNJune20/Assignment22/PalindromeString.java (hunk 1 above)
Comment (VireshJJun20) [reply src/suparnaNJune20/Assignment22/PalindromeString.java:33]: Another method can be simply reverse the input string and then compare it with original one.
number of coding lines will also get decreased. you can try if you want. :)

**Review by VireshJJun20 — CHANGES_REQUESTED** [review]
Approved

**Review by VireshJJun20 — APPROVED** [review]

File: src/suparnaNJune20/Assignment22/PalindromeNo.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+		int tempNum = 0, orignalNum = num;
```
Comment (KrishnaKTechnocredits) [inline src/suparnaNJune20/Assignment22/PalindromeNo.java:18]: The variable name should be revNumber, not tempNumber.

File: src/suparnaNJune20/Assignment22/PalindromeString.java [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+		int lIndex = str1.length() - 1;
+		int fIndex = 0;
+		char ch1, ch2;
+		if (lIndex / 2 != 0) {
```
Comment (KrishnaKTechnocredits) [inline src/suparnaNJune20/Assignment22/PalindromeString.java:37]: This logic is not acceptable. Always simply the logic.

**Pull request merged**